apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -69,10 +97,36 @@ public abstract partial class BigtableClient
         /// "default" application profile will be used.
         /// </param>
         /// <returns>The created <see cref="BigtableClient"/>.</returns>
-        public static BigtableClient Create(ServiceEndpoint endpoint = null, BigtableServiceApiSettings settings = null, string appProfileId = null)
+        public static BigtableClient Create(BigtableServiceApiSettings settings = null, ClientCreationSettings clientCreationSettings = null, string appProfileId = null)
         {
-            var client = BigtableServiceApiClient.Create(endpoint, settings);
-            return new BigtableClientImpl(client, appProfileId) { UnderlyingClientSettings = settings };
+            clientCreationSettings?.Validate();
+            var channelCredentials = clientCreationSettings?.Credentials;
+
+            // Use default credentials if none given.
+            if (channelCredentials == null)
+            {
+                var credentials = GoogleCredential.GetApplicationDefault();
+                if (credentials.IsCreateScopedRequired)
+                {
+                    credentials = credentials.CreateScoped(BigtableServiceApiClient.DefaultScopes);
+                }
+                channelCredentials = credentials.ToChannelCredentials();
+            }
+            var clientCount = clientCreationSettings?.ClientCount ?? Environment.ProcessorCount;
+
+            var endpoint = clientCreationSettings?.ServiceEndpoint ?? BigtableServiceApiClient.DefaultEndpoint;
+            var clients = new BigtableServiceApiClient[clientCount];
+            var shutdowns = new Func<Task>[clientCount];
+            // Fill clients[] with BigtableServiceApiClient instances, each with specific channel
+            for (int i = 0; i < clientCount; i++)
+            {
+                var channel = new Channel(endpoint.Host, endpoint.Port, channelCredentials);
+                clients[i] = BigtableServiceApiClient.Create(channel, clientCreationSettings?.BigtableServiceApiSettings);
+                shutdowns[i] = channel.ShutdownAsync;
+            }
+            Func<Task> shutdown = () => Task.WhenAll(shutdowns.Select(x => x()));
+
+            return new BigtableClientImpl(appProfileId, clients) { UnderlyingClientSettings = settings };
```

Comment:
Rather than duplicate everything here, it would be better to make this whole method an expression-bodied member that delegates to the async version:
```csharp
=> Task.Run(() => CreateAsync(clientCreationSettings, appProfileId)).ResultWithUnwrappedExceptions();
```